docs/wiki/api-references/Base-Nodes-API.md

```diff
@@ -1,112 +1,41 @@
-
-## `const`
-
-```python
-csp.const(
-    value: '~T',
-    delay: timedelta = timedelta()
-)
-```
-
-This will create an edge that ticks one time with the value provided.
-By default this will tick at the start of the engine, delta can be provided to delay the tick
-
-## `curve`
-
-```python
-csp.curve(
-    typ: 'T',
-    data: typing.Union[list, tuple]
-)
-```
-
-This allows you to convert a list of non-csp data into a ticking edge in csp
-
-Args:
-
-- **`typ`**: is the type of the value of the data of this edge
-- **`data`**: is either a list of tuples of `(datetime, value)`, or a tuple of two equal-length numpy ndarrays, the first with datetimes and the second with values.
-  In either case, that will tick on the returned edge into the engine, and the data must be in time order.
-  Note that for the list of tuples case, you can also provide tuples of (timedelta, value) where timedelta will be the offset from the engine's start time.
-
-## `add_graph_output`
-
-```python
-csp.add_graph_output(
-    key: object,
-    input: ts['T'],
-    tick_count: int = -1,
-    tick_history: timedelta = timedelta()
-)
-```
-
-This allows you to connect an edge as a "graph output".
-All edges added as outputs will be returned to the caller from `csp.run` as a dictionary of `key: [(datetime, value)]`
-(list of datetime, values that ticked on the edge) or if `csp.run` is passed `output_numpy=True`, as a dictionary of
-`key: (array, array)` (tuple of two numpy arrays, one with datetimes and one with values).
-See [Collecting Graph Outputs](https://github.com/Point72/csp/wiki/0.-Introduction#collecting-graph-outputs)
-
-Args:
-
-- **`key`**: key to return the results as from csp.run
-- **`input`**: edge to connect
-- **`tick_count`**: number of ticks to keep in the buffer (defaults to -1 - all ticks)
-- **`tick_history`**: amount of ticks to keep by time window (defaults to keeping all history)
-
-## `feedback`
-
-```python
-csp.feedback(typ)
-```
-
-`csp.feedback` is a construct that can be used to create artificial loops in the graph.
-Use feedbacks in order to delay bind an input to a node in order to be able to create a loop
-(think of writing a simulated exchange that takes orders in and needs to feed responses back to the originating node).
-
-`csp.feedback` itself is not an edge, its a construct that allows you to access the delayed edge / bind a delayed input.
-
-Args:
-
-- **`typ`**: type of the edge's data to be bound
+The latest set of base nodes can be found in the `csp.baselib` module.
 
-Methods:
+All of the nodes noted here are imported directly into the csp namespace when importing csp.
 
-- **`out()`**: call this method on the feedback object to get the edge which can be wired as an input
-- **`bind(x: ts[object])`**: call this to bind an edge to the feedback
+These are all graph-time constructs.
 
-# Basic Nodes
+## Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [`print`](#print)
```

Comment:
I think we should put the full name of the methods here with the "csp." prefix

Reply:
why dont I see this resolved?

Reply:
This was hidden for me, updated this and the base-adapters reference pages now